development.ini

```diff
@@ -15,7 +14,10 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_tm
 
-sqlalchemy.url = postgresql+psycopg2://jackbot:@localhost:5432/
+jinja2.filters =
+;    datetimeformat = journalapp.jinja2_filters.datetimeformat
+;    dateformat = journalapp.jinja2_filters.dateformat
+    markdown = markdown.markdown
```

Comment:
This should work, but on it's own it won't handle things like fenced code blocks or colorizing code samples.